### DISCUSSION TITLE
Copy as image - loss of labels

### DISCUSSION THREAD

**Discussion by peterknicol1951** [wxMaxima-developers/wxmaxima#1579] [General]
Hello. In previous versions my students were able to use Copy as Image to display their wxMaxima work **including the labels %i1, %o1** etc. This useful function does not appear to work now. I have attached a screenshot of the original and an image of the resulting pasted image. Is there any way of enabling the inclusion of the labels?
Any help will be appreciated,
Peter

Original is 
%i1 3*x^2
 %o1 3x2  (2 a superscript) Can't paste a screenshot here
Pasted image below:
![image](https://user-images.githubusercontent.com/35968161/132332702-3da7725b-7d55-4dd0-9681-1348ff01053b.png)

Details of program:
![image](https://user-images.githubusercontent.com/35968161/132332324-0734ca1b-f86c-418f-9150-a8884014acca.png)

**Comment by peterknicol1951** [comment]
Is this issue #1507 ?

**Comment by peterknicol1951** [comment]
Further build info from MS Windows machine with same issue:
![BUILD-INFO (M  PATEL)](https://user-images.githubusercontent.com/35968161/135911099-cb7c0f08-4ddc-45b0-9ed1-b0a7b443038a.PNG)

**Comment by yurytch** [comment]
Solution suggested in #1507
